server/package.json

```diff
@@ -6,6 +6,7 @@
   "private": true,
   "scripts": {
     "dev": "nodemon -w src --exec \"babel-node src\"",
+    "debug": "nodemon -w src src/index.js --exec 'babel-node --inspect'",
```

Comment:
❤️

Reply:
Cool to have it, as for future: let's do infra changes in a separate task for easier review and testing 😉

Reply:
@AleksSenkou you're right, but I wasn't confident that I can fix bug without it, so added here 😉